sdk/monitor/Azure.Monitor.Ingestion/api/Azure.Monitor.Ingestion.netstandard2.0.cs

```diff
@@ -19,11 +19,20 @@ public enum ServiceVersion
             V2021_11_01_Preview = 1,
         }
     }
+    public partial class UploadFailedArgs : Azure.SyncAsyncEventArgs
```

Comment:
You need `EventArgs` in the name.  See the derived classes at https://learn.microsoft.com/en-us/dotnet/api/system.eventargs?view=net-7.0 for comparison.  `UploadFailedEventArgs` would be better.